[CI] BulkAfterWriteFsyncFailureIT testFsyncFailureDoesNotAdvanceLocalCheckpoints failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #3284 / amazonlinux-2023_platform-support-aws](https://gradle-enterprise.elastic.co/s/cjetwi5xvffxa)
- [elasticsearch-periodic-platform-support #3280 / debian-11_platform-support-unix](https://gradle-enterprise.elastic.co/s/t7caaw635rvu4)
- [elasticsearch-periodic-platform-support #3272 / centos-7_platform-support-unix](https://gradle-enterprise.elastic.co/s/jqqqw4cpzr654)

**Reproduction Line:**
```
./gradlew ':server:internalClusterTest' --tests "org.elasticsearch.action.bulk.BulkAfterWriteFsyncFailureIT.testFsyncFailureDoesNotAdvanceLocalCheckpoints" -Dtests.seed=D492149CFC74C8EC -Dtests.locale=sl-SI -Dtests.timezone=Etc/GMT+5 -Druntime.java=22
```

**Applicable branches:**
8.15

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('testFsyncFailureDoesNotAdvanceLocalCheckpoints'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.action.bulk.BulkAfterWriteFsyncFailureIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
java.lang.AssertionError: null
```

**Issue Reasons:**
- [8.15] 3 failures in test testFsyncFailureDoesNotAdvanceLocalCheckpoints (2.7% fail rate)
- [8.15] 3 failures in pipeline elasticsearch-periodic-platform-support (75.0% fail rate)
- [8.15] 2.7% failure rate in test testFsyncFailureDoesNotAdvanceLocalCheckpoints (2.7% fail rate)

**Note:**
This issue was created using new, in-development muting automation. Please mind any issues.

--- a/server/src/internalClusterTest/java/org/elasticsearch/action/bulk/BulkAfterWriteFsyncFailureIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/action/bulk/BulkAfterWriteFsyncFailureIT.java
@@ -29,6 +29,7 @@ import java.util.Set;
 import java.util.concurrent.atomic.AtomicBoolean;
 
 import static org.elasticsearch.index.IndexSettings.INDEX_REFRESH_INTERVAL_SETTING;
+import static org.elasticsearch.indices.IndicesService.WRITE_DANGLING_INDICES_INFO_SETTING;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.greaterThan;
 import static org.hamcrest.Matchers.is;
@@ -48,7 +49,11 @@ public class BulkAfterWriteFsyncFailureIT extends ESSingleNodeTestCase {
         PathUtilsForTesting.teardown();
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/110551")
+    @Override
+    protected Settings nodeSettings() {
+        return Settings.builder().put(WRITE_DANGLING_INDICES_INFO_SETTING.getKey(), false).build();
+    }
+
     public void testFsyncFailureDoesNotAdvanceLocalCheckpoints() {
         String indexName = randomIdentifier();
         client().admin()

